### ISSUE TITLE
Unstructured data fitting

### ISSUE THREAD

**Issue by sixpearls** [kb-press/ndsplines#33]

**Comment by sixpearls** [comment]
@ixjlyons What do you think about the scope of this for the first release/publication?

**Comment by ixjlyons** [comment]
I agree it seems out of scope and/or a good candidate for future work.

> the scikit-learn package inexplicably renamed the algorithm to EARTH

Curiosity got the best of me. Apparently MARS is trademarked so they chose the nearest planet to it<sup>1</sup> as an alternative. Seems like the original was an R implementation.

<sup><sup>1</sup>Made that part up but seems plausible</sup>
